source/cache-proxy/setup-squid-as-a-transparent-proxy.rst

```diff
@@ -45,16 +47,18 @@ configured.
       is operational and tested.
    #. Click Save
 
-#. Click on the **Cache Management** tab.
+#. Click on the **Local Cache** tab.
 
-   #. Hard disk cache size: Set this as needed, but keep it a reasonable
+   #. Hard disk cache size (in MB): Set this as needed, but keep it a reasonable
       size. 3000 (3GB) may be a good place to start.
    #. Hard disk cache location: Should be */var/squid/cache* but may be
       moved if needed
    #. Memory cache size: The amount of RAM that squid should claim for
       caching. Use as much as can be spared, as this is much faster than
       caching to disk. It should not exceed 50% of the installed RAM,
       however.
+   #. Hard disk cache location: The directory where the cache will be stored.
+      If using a non-default location (for example, if you have added a separate drive for caching), enter it here.
```

Comment:
Avoid addressing the reader here, reword that sentence to remove "you", such as: "for example, if the firewall utilizes a separate drive for caching"